mtrf/matrices.py

```diff
@@ -132,24 +132,19 @@ def covariance_matrices(x, y, lags, zeropad=True, preload=True):
     x, y, _ = _check_data(x, y)
     if zeropad is False:
         y = truncate(y, lags[0], lags[-1])
-
-    x_lag_size = x[0].shape[-1] * len(lags) + 1
```

Comment:
Still believe move the if..else branch outside will make the code more efficient (to avoid repeatedly checking the same value). Calculating the covariance matrix is one of the core functionality of this package, we should keep it as efficient as we can.